src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -211,6 +211,7 @@ internal class OptionsGump : Gump
         private Checkbox _use_tooltip;
         private Checkbox _useStandardSkillsGump, _showMobileNameIncoming, _showCorpseNameIncoming;
 
+        Profile currentProfile = ProfileManager.CurrentProfile;
```

Comment:
missing the `private` access modifier and the name convention is wrong: missing the “_” before the name

Reply:
Fixed.